Simplists/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleVersion</key>
-	<string>57978332</string>
+	<string>1</string>
```

Comment:
Removed build script that was setting build number because it wasn't working for new widget & intent extensions.  Those stubbornly wanted to use `$CURRENT_PROJECT_VERSION` from project file.  Chose to update that setting and not automate setting of build number for this release.  Will revisit later.